phpstan.dist.neon

```diff
@@ -3,6 +3,10 @@ includes:
     - dev-tools/vendor/phpstan/phpstan/conf/bleedingEdge.neon
     - dev-tools/vendor/phpstan/phpstan-phpunit/extension.neon
 
+    # Baseline, should only shrink! To regenerate it, just execute:
+    # dev-tools/vendor/bin/phpstan analyse --generate-baseline=dev-tools/phpstan/baseline.php
```

Comment:
question: why in php and not in neon format?

Reply:
It allows simpler operations when you need to programatically analyse baseline file - you only need to do `$errors = require 'dev-tools/phpstan/baseline.php';` and you don't need additional libraries to parse non-native format. Also, paths in PHP baseline should be clickable and should navigate to files containing error - in theory, because somehow PHPStorm has problems with it currently (it treats `__DIR__` as project's root dir so it can't resolve path for baselines stored in subdirectories).

Reply:
> when you need to programatically analyse baseline file - you only need to do `$errors = require 'dev-tools/phpstan/baseline.php';`

do we need it, indeed? when? why?

Reply:
Currently we don't need it. I used it mostly for file navigation, [hoping it can be fixed](https://twitter.com/_Codito_/status/1642899049547280387) 🤷‍♂️.

Reply:
I forgot to mention one fundamental argument - it provides [performance gain](https://mobile.twitter.com/OndrejMirtes/status/1628766083514507265), especially for large baselines.

Reply:
twitter-based docs 😅

Reply:
FYI: there's [issue in the bugtracker](https://youtrack.jetbrains.com/issue/WI-72092/Completion-in-file-path-injection-stops-working-after-reaching-a-certain-length), and it's a regression, so maybe they'll fix it soon and then it will be usable 🙂.